startup/gui/menus.py

```
@@ -554,6 +554,17 @@ def __usdLightCreator( lightType ) :
 nodeMenu.append( "/Dispatch/Frame Mask", GafferDispatch.FrameMask, searchText = "FrameMask" )
 nodeMenu.append( "/Dispatch/Local Dispatcher", GafferDispatch.LocalDispatcher, searchText = "LocalDispatcher" )
 
+# ML nodes
+
+import GafferML
+import GafferMLUI
```

Ruff

`GafferMLUI` imported but unused

Remove unused import: `GafferMLUI`

(F401)